Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM phusion/baseimage:0.10.1
+FROM ubuntu:bionic
```

Comment:
We're not using stock ubuntu images for a reason. See https://phusion.github.io/baseimage-docker/ for explanations.
phusion/baseimage:0.11 is based on Ubuntu-18.04 - did you have a specific reason to move away from phusion?

Reply:
Agree on keeping phusion by changing to `0.11` if this gets the curl issue fixed. Thanks for the explications @pmconrad and sorry for not noticing.

Reply:
Will rebuild using phusion 0.11, if everything goes fine and starts with no issues, i'll push the changes, thanks a lot for the input guys

Reply:
Will rebuild using phusion 0.11, if everything goes fine and starts with no issues, i'll push the changes, thanks a lot for the input guys

Reply:
Builds/Runs succesfully on 0.11, pushed code